sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -24,13 +24,13 @@ package org.apache.spark.sql.catalyst.expressions.codegen
  * Written by Matei Zaharia.
  */
 object CodeFormatter {
-  def format(code: String): String = new CodeFormatter().addLines(code).result()
+  def format(code: String): String = new CodeFormatter().addLines(stripExtraNewLines(code)).result()
```

Comment:
Thank you for review, @davies .
Oh, I thought `CodeFormatter.format` is called before Janino and Guava loading cache, too.
I'll make that consistent in this afternoon. If then, it'll be okay.